src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+  justify-content: space-between;
+
+  &__moyo_logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+  justify-content: flex-end;
+  padding-left: 0;
```

Comment:
The `padding-left: 0;` is set for both `.nav` and `.nav__list`, which might be redundant. Consider removing it from one of these selectors if it's not necessary.